gax/src/main/java/com/google/api/gax/batching/BatcherImpl.java

```diff
@@ -225,7 +225,7 @@ public void close() throws InterruptedException {
    */
   private static class Batch<ElementT, ElementResultT, RequestT, ResponseT> {
     private final BatchingRequestBuilder<ElementT, RequestT> builder;
-    private final List<SettableApiFuture<ElementResultT>> results;
```

Comment:
Should this variable be renamed to entries?

Reply:
Yes, It should have been. Updated.